manual/sphinx/user_docs/installing.rst

```diff
@@ -130,6 +130,10 @@ The bare-minimum requirements for compiling and running BOUT++ are:
    
 The FFTW-3 library ( `http://www.fftw.org/ <http://www.fftw.org/>`__ ) is also strongly recommended
```

Comment:
Maybe make it clear that it is not requiered, but some features will be missing?
Though I am happy to keep as is

Reply:
Thanks, fixed